modules/custom-tuning-specification.adoc

```diff
@@ -0,0 +1,136 @@
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+See the link:https://access.redhat.com/support/offerings/techpreview/[Red Hat
+Technology Preview features support scope] for more information.
+endif::[]
+====
+
+The custom resource (CR) for the operator has two major sections. The first
+section, `profile:`, is a list of tuned profiles and their names. The second,
+`recommend:`, defines the profile selection logic.
+
+Multiple custom tuning specifications can co-exist as multiple CRs in the
+operator's namespace. The existence of new CRs or the deletion of old CRs is
+detected by the Operator. All existing custom tuning specifications are merged
+and appropriate objects for the containerized tuned daemons are updated.
+
+*profile: data*
+
+The `profile:` section lists tuned profiles and their names.
+
+----
+profile:
+- name: tuned_profile_1
+  data: |
+    # Tuned profile specification
+    [main]
+    summary=Description of tuned_profile_1 profile
+
+    [sysctl]
+    net.ipv4.ip_forward=1
+    # ... other sysctl's or other tuned daemon plugins supported by the containerized tuned
+
+# ...
+
+- name: tuned_profile_n
+  data: |
+    # Tuned profile specification
+    [main]
+    summary=Description of tuned_profile_n profile
+
+    # tuned_profile_n profile settings
+----
+
+*Recommended profiles*
+
+The `profile:`selection logic is defined by the recommend section of the CR:
+
+----
+recommend:
+- match:                              # optional; if omitted, profile match is assumed unless a profile with a higher matches first
+  <match>                             # an optional array
+  priority: <priority>                # profile ordering priority, lower numbers mean higher priority (0 is the highest priority)
+  profile: <tuned_profile_name>       # e.g. tuned_profile_1
+
+# ...
+
+- match:
+  <match>
+  priority: <priority>
+  profile: <tuned_profile_name>       # e.g. tuned_profile_n
+----
+
+If `<match>` is omitted, a profile match (for example, `true`) is assumed.
+
+`<match>` is an optional array recursively defined as follows:
+
+----
+- label: <label_name>     # node or pod label name
+  value: <label_value>    # optional node or pod label value; if omitted, the presence of <label_name> is enough to match
+  type: <label_type>      # optional node or pod type ("node" or "pod"); if omitted, "node" is assumed
+  <match>                 # an optional <match> array
+----
+
+If `<match>` is not omitted, all nested `<match>` sections must also evaluate to
+`true`. Otherwise, `false` is assumed and the profile with the respective
+`<match>` section will not be applied or recommended. Therefore, the nesting
+(child `<match>` sections) works as logical and Operator. Conversely, if any
+item of the `<match>` array matches, the entire `<match>` array evaluates to
+`true`. Therefore, the array acts as logical or Operator.
```

Comment:
Need to make sure _or_ is a distinguished as a logical *o*perator or just replace the logical operator with _connective_ to avoid confusion.